x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/ip/IpPrefixTests.java

```diff
@@ -106,7 +106,7 @@ public static Iterable<Object[]> parameters() {
             })
         );
 
-        return parameterSuppliersFromTypedData(errorsForCasesWithoutExamples(anyNullIsNull(true, suppliers)));
+        return parameterSuppliersFromTypedData(errorsForCasesWithoutExamples(anyNullIsNull(true, randomizeBytesRefsOffset(suppliers))));
```

Comment:
This chain is starting to (And will) get longer over time I think. Should we replace it with a "recommendedModifiers()"? Specially for average cases where all of them are applied.
Maybe not for this PR though, as this is a fix